packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -44,7 +55,14 @@ export const BundleMenuButton = ({bundle}: Props) => {
       id="bundle-menu"
       menu={
         <Menu>
-          <MenuItem onClick={handleOnDeleteBundle} icon={TrashIcon} text="Delete release" />
+          <MenuItem
+            onClick={handleOnToggleArchive}
+            // TODO: swap line once UnaryIcon is available
```

Comment:
I can do this as a fast-follow once sanity-icons has been updated with this icon